src/net/sourceforge/kolmafia/textui/command/TestCommand.java

```diff
@@ -200,6 +200,7 @@ public void run(final String cmd, final String parameters) {
         RequestLogger.printLine("Inside try: checkpoint known = " + saved.known());
       }
       RequestLogger.printLine("Outside try: checkpoint known = " + saved.known());
+      saved.close();
```

Comment:
is this one a false positive? if we try-with-resources `checkpoint`, then set `saved` to `checkpoint`, `saved` is always going to be closed after it exits the block?

I'm fine with the change to suppress the warning though... :/

Reply:
ah yeah i'll remove it